snaploader/src/main/java/electrostatic4j/snaploader/platform/util/NativeVariant.java

```diff
@@ -81,6 +91,15 @@ public enum NativeVariant {
 
     private final String property;
 
+    /**
+     * named CPU features that were detected by the OSHI library
+     */
+    private static Collection<String> presentFeatures;
+    /**
+     * serialize access to presentFeatures
+     */
+    private static Object synchronizeFeatures = new Object();
+
```

Comment:
For readability purposes, I suggest to move these objects to `NativeVariant.Cpu` class.

Reply:
Done.